content/applications/sales/sales/sales_quotations.rst

```diff
@@ -0,0 +1,138 @@
+
+In Odoo, quotations are configured in the **Sales** app. They can also be generated from other apps
+as part of the sales workflow:
+
+- CRM: :doc:`Convert opportunities <../crm/acquire_leads/send_quotes>` into quotes to follow up on
+  potential deals.
+- Helpdesk: :doc:`Generate quotes from tickets <../../services/helpdesk/advanced/after_sales>` when
+  offering paid services or products.
+- Subscriptions: :doc:`Offer recurring services <../subscriptions>` before
+  starting an automatic billing cycle.
+
+.. cards::
+
+   .. card:: Create quotations
+      :target: sales_quotations/create_quotations
+
+      Create, configure, and send quotations to customers.
+
+   .. card:: Quotation templates
+      :target: sales_quotations/quote_template
+
+      Configure and use quotation templates to send tailor-fit quotations at a quicker pace.
+
+   .. card:: Optional products
+      :target: sales_quotations/optional_products
+
+      Offer useful and related products to customers to increase sales.
+
+   .. card:: Online signatures for order confirmations
+      :target: sales_quotations/get_signature_to_validate
+
+      Customers have the ability to confirm orders via online signatures, directly on sales orders.
+
+   .. card:: Online payment order confirmation
+      :target: sales_quotations/get_paid_to_validate
+
+      Customers have the ability to confirm orders via online payment, directly on sales orders.
+
+   .. card:: Quotation Deadlines
+      :target: sales_quotations/deadline
+
+      Set deadlines on quotations to encourage customers to act in a timely manner when
+      closing business deals.
+
+   .. card:: Deliver orders and invoices to different addresses
+      :target: sales_quotations/different_addresses
+
+      Specify separate customer delivery and invoicing addresses on quotations.
+
+   .. card:: Product variants on quotations and sales orders
+      :target: sales_quotations/orders_and_variants
+
+      Add product variants to sales orders to provide additional options for single products.
+
+   .. card:: PDF quote builder
+      :target: sales_quotations/pdf_quote_builder
+
+      Add custom PDF files to quotations to elevate the document's headers and designs.
+
+Sales quotations in business deals
+==================================
+
+Sales quotations serve as a key step in the sales process, bridging the gap between a customer's
+initial inquiry about goods and services and the final contractual agreement for payment and
+delivery. The quotation also provides transparency in pricing, helping both parties negotiate and
+finalize the terms before making a commitment.
+
+Sales quotations play a crucial role in business transactions by defining the scope and cost of what
+is being sold to the end-customer; setting clear expectations on pricing, delivery, tax, and
+payment terms; and providing a documented stage where the business deal can be negotiated before
+agreed upon.
+
+Key components of a sales quotation
```

Comment:
I love this section, this is an excellent example of how to "show" (vs. tell) and lead with a strong success model both in written and visual form. Very nice!